prime-dotnet-webapi/Controllers/EnrolleesController.cs

```diff
@@ -54,18 +54,18 @@ public class EnrolleesController : PrimeControllerBase
         [Authorize(Roles = Roles.ViewEnrollee)]
         [ProducesResponseType(typeof(ApiMessageResponse), StatusCodes.Status400BadRequest)]
         [ProducesResponseType(StatusCodes.Status401Unauthorized)]
-        [ProducesResponseType(typeof(ApiResultResponse<IEnumerable<EnrolleeListViewModel>>), StatusCodes.Status200OK)]
+        [ProducesResponseType(typeof(ApiResultResponse<PaginatedList<EnrolleeListViewModel>>), StatusCodes.Status200OK)]
```

Comment:
I think this should be
`[ProducesResponseType(typeof(ApiResultResponse<PaginatedResponse<EnrolleeListViewModel>>), StatusCodes.Status200OK)]`